app/Livewire/Project/Application/General.php

```
@@ -471,4 +520,85 @@ public function downloadConfig()
+
+    private function updateServiceEnvironmentVariables()
+    {
+        $domains = collect(json_decode($this->application->docker_compose_domains, true)) ?? collect([]);
+
+        foreach ($domains as $serviceName => $service) {
+            $serviceNameFormatted = str($serviceName)->upper()->replace('-', '_');
+            $domain = data_get($service, 'domain');
+
+            if ($domain) {
+                // Create or update SERVICE_FQDN_ and SERVICE_URL_ variables
+                $fqdn = Url::fromString($domain);
```

PHPMD (2.15.0)

Avoid using static access to class '\Spatie\Url\Url' in method 'updateServiceEnvironmentVariables'. (Clean Code Rules)

(StaticAccess)